tests/sync_tests/test_http2.py

```diff
@@ -247,164 +247,3 @@ def test_request_with_missing_host_header() -> None:
-#         )
-#         status_code, headers, stream, extensions = response
-#         body = stream.read()
-#         assert status_code == 200
-#         assert body == b"Hello, world."
-#         assert http.get_connection_info() == {
-#             "http://example.org": ["HTTP/1.1, IDLE"]
-#         }
-#
-#         # On sending this second request, at the point of pool re-acquiry the
-#         # socket indicates that it has disconnected, and we'll send the request
-#         # over a new connection.
-#         response = http.handle_request(
-#             method=b"GET",
-#             url=(b"http", b"example.org", None, b"/"),
-#             headers=[(b"Host", b"example.org")],
-#             stream=httpcore.ByteStream(b""),
-#             extensions={},
-#         )
-#         status_code, headers, stream, extensions = response
-#         body = stream.read()
-#         assert status_code == 200
-#         assert body == b"Hello, world."
-#         assert http.get_connection_info() == {
-#             "http://example.org": ["HTTP/1.1, IDLE"]
-#         }
-#
-#
-# @pytest.mark.trio
-# def test_get_request_with_unclean_close_after_first_request() -> None:
-#     backend = MockBackend(
-#         http_buffer=[
-#             b"HTTP/1.1 200 OK\r\n",
-#             b"Date: Sat, 06 Oct 2049 12:34:56 GMT\r\n",
-#             b"Server: Apache\r\n",
-#             b"Content-Length: 13\r\n",
-#             b"Content-Type: text/plain\r\n",
-#             b"\r\n",
-#             b"Hello, world.",
-#             b"",  # Terminate the connection.
-#         ],
-#     )
-#
-#     with httpcore.SyncConnectionPool(backend=backend) as http:
-#         # Send an initial request. We're using a standard keep-alive
-#         # connection, so the connection remains in the pool after completion.
-#         response = http.handle_request(
-#             method=b"GET",
-#             url=(b"http", b"example.org", None, b"/"),
-#             headers=[(b"Host", b"example.org")],
-#             stream=httpcore.ByteStream(b""),
-#             extensions={},
-#         )
-#         status_code, headers, stream, extensions = response
-#         body = stream.read()
-#         assert status_code == 200
-#         assert body == b"Hello, world."
-#         assert http.get_connection_info() == {
-#             "http://example.org": ["HTTP/1.1, IDLE"]
-#         }
-#
-#         # At this point we successfully write another request, but the socket
-#         # read returns `b""`, indicating a premature close.
-#         with pytest.raises(httpcore.RemoteProtocolError) as excinfo:
-#             http.handle_request(
-#                 method=b"GET",
-#                 url=(b"http", b"example.org", None, b"/"),
-#                 headers=[(b"Host", b"example.org")],
-#                 stream=httpcore.ByteStream(b""),
-#                 extensions={},
-#             )
-#         assert str(excinfo.value) == "Server disconnected without sending a response."
```

Comment:
Not sure how this leaked in, but it's my fault it's there.
We might sometime deal with this unrelated cleanup in a separately targeted PR, but let's just roll with it here.
Keep things simple, keep things moving.

Reply:
sure, raise a new issue itll be cool to help